src/Compilers/VisualBasic/Test/Semantic/Binding/BindingCollectionInitializerTests.vb

```diff
@@ -701,10 +701,14 @@ IObjectOrCollectionInitializerExpression (OperationKind.ObjectOrCollectionInitia
       IInvalidExpression (OperationKind.InvalidExpression, Type: System.Void, IsInvalid) (Syntax: '"Hello World!"')
         Children(2):
             IOperation:  (OperationKind.None, IsInvalid) (Syntax: '"Hello World!"')
+              Children(1):
+                  IOperation:  (OperationKind.None, IsInvalid) (Syntax: 'New C2() Fr ... r element"}')
```

Comment:
>IOperation:  (OperationKind.None, IsInvalid) (Syntax: 'New C2() Fr ... r element"}') [](start = 18, length = 84)

This node probably should be marked as Implicit, please open an issue to follow up.